src/function/table/CMakeLists.txt

```diff
@@ -1,11 +1,13 @@
 add_subdirectory(sqlite)
-add_library_unity(duckdb_func_table
-                  OBJECT
-                  range.cpp
-                  repeat.cpp
-                  copy_csv.cpp
-                  read_csv.cpp
-                  sqlite_functions.cpp)
+add_library_unity(
```

Comment:
This change is the result of running `make format`. 

I'm ok with not keeping cosmetic changes in a PR, however the next person who runs `make format` on this file, won't also have to include this change if we keep it.